### PR TITLE
test/e2e: Cleanup and suffix namespace

### PR DESCRIPTION
**Pull request by brancz**
As a follow up of https://github.com/openshift/cluster-monitoring-operator/pull/262#discussion_r

@mxinden @s-urbaniak

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -709,6 +709,21 @@ func (c *Client) CreateOrUpdateNamespace(n *v1.Namespace) error {
...
+	if apierrors.IsNotFound(err) {
+		// Namespace already deleted
+		return nil
+	}
+	if err != nil {
+		return errors.Wrap(err, "retrieving Namespace object failed")
+	}
+
+	err = nClient.Delete(nsName, &metav1.DeleteOptions{})
+	return errors.Wrap(err, "deleting ConfigMap object failed")
```
Comment (mxinden) [inline pkg/client/client.go]: `s/ConfigMap/Namespace`

File: pkg/client/client.go [hunk 2]
```diff
@@ -709,6 +709,21 @@ func (c *Client) CreateOrUpdateNamespace(n *v1.Namespace) error {
 	return errors.Wrap(err, "updating ConfigMap object failed")
 }
 
+func (c *Client) DeleteIfExists(nsName string) error {
```
Comment (mxinden) [inline pkg/client/client.go]: `s/DeleteIfExists/DeleteNSIfExists`

**Comment by brancz** [comment]
Whops, sorry about that, please go ahead @mxinden!

**Pull request merged**